app/src/org/commcare/tasks/ModernHttpTask.java

```diff
@@ -107,4 +114,9 @@ protected void deliverError(HttpResponseProcessor httpResponseProcessor,
     public InputStream getResponseStream() {
         return responseDataStream;
     }
+
+    @Override
+    public String getApiVersion() {
+        return apiVersion;
```

Comment:
how about `return requester.getApiVersion()` ? 

Also is this API version connect only or will also be added to non connect server calls ?

Reply:
Ah, excellent suggestion, I'll do that!

Currently both the ConnectId and Connect servers are returning this header. I'm not sure about any plans for HQ to use it.